src/renderer/reader/components/ReaderMenu.tsx

```diff
@@ -227,13 +227,13 @@ export class ReaderMenu extends React.Component<IProps, IState> {
                                     isRTL ? styles.rtlDir : " ")
                             }
                             onClick=
-                                {link.Href ? (e) => this.props.handleLinkClick(e, link.Href) : undefined}
+                                {link.Href ? (e) => this.props.handleLinkClick(e, link.Href, false) : undefined}
```

Comment:
Post-merge comment: I disagree with this breaking UX change. In fact, this PR implementation goes against @llemeurfr 's own suggestion:

> I'd personally prefer to keep the default behavior as-is (one click on a ToC entry or return key means move to the entry and close the panel)

https://github.com/edrlab/thorium-reader/issues/1449#issuecomment-848647409

Reply:
Follow-up PR: https://github.com/edrlab/thorium-reader/pull/1484